### PR TITLE
Update ROS 2 distros in README.md

### PR DESCRIPTION
**Pull request by sea-bass**
**Public API Changes**
N/A

**Description**
This PR updates the ROS 2 distros displayed in the repo's top-level README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,10 @@ Releasing requires push access to [RobotWebTools/rosbridge_suite](https://github
 2. Manually edit and clean up the changelogs. Commit the changes.
 3. Run `catkin_prepare_release --bump [major/minor/patch]` to bump versions in package.xml and push changes to origin.
 4. Run bloom-release commands to create PRs to update rosdistro:
-    - `bloom-release --rosdistro foxy --track foxy rosbridge_suite`
-    - `bloom-release --rosdistro galactic --track galactic rosbridge_suite`
-    - `bloom-release --rosdistro rolling --track rolling rosbridge_suite`
+    - `bloom-release --rosdistro humble rosbridge_suite`
+    - `bloom-release --rosdistro humble rosbridge_suite`
```
Comment (sea-bass) [inline README.md]:
```suggestion
    - `bloom-release --rosdistro iron rosbridge_suite`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 rosbridge_suite
 ===============
 
-[![ROS Foxy version](https://img.shields.io/ros/v/foxy/rosbridge_suite)](https://index.ros.org/p/rosbridge_suite/github-RobotWebTools-rosbridge_suite/#foxy)
-[![ROS Galactic version](https://img.shields.io/ros/v/galactic/rosbridge_suite)](https://index.ros.org/p/rosbridge_suite/github-RobotWebTools-rosbridge_suite/#galactic)
 [![ROS Humble version](https://img.shields.io/ros/v/humble/rosbridge_suite)](https://index.ros.org/p/rosbridge_suite/github-RobotWebTools-rosbridge_suite/#humble)
+[![ROS Iron version](https://img.shields.io/ros/v/humble/rosbridge_suite)](https://index.ros.org/p/rosbridge_suite/github-RobotWebTools-rosbridge_suite/#iron)
+[![ROS Jazzy version](https://img.shields.io/ros/v/humble/rosbridge_suite)](https://index.ros.org/p/rosbridge_suite/github-RobotWebTools-rosbridge_suite/#jazzy)
```
Comment (bjsowa) [inline README.md]: The images point to humble

**Review by bjsowa — CHANGES_REQUESTED** [review]
Since we branched out, it may be worth mentioning the development branch for each distro (`humble` for ROS Humble, `ros2` for Iron, Jazzy and Rolling)

File: README.md (hunk 2 above)
Comment (sea-bass) [reply README.md]: thanks, I missed some... fixed it!

**Review by bjsowa — APPROVED** [review]

**Pull request merged**